portfolio/src/main/webapp/js/pets.js

```diff
@@ -0,0 +1,12 @@
+function slideShow() {
+    setTimeout(slideShow, 1000);
+    const img = document.querySelectorAll(".image");
+    for(var i = 0; i < 7; i++) {
```

Comment:
instead of just using 7 here, can you use the number of images in the page? the way you have this written now, if you added an 8th guinea pig photo (or removed one), this would break.

Reply:
Good practice to not hard code! Done.